src/css/homepage.css

```diff
@@ -0,0 +1,43 @@
+.events-section {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  margin: 0;
+}
+
+.event-item {
+  position: relative;
+  width: calc(50% - 40px);
+  min-height: 140px;
+  margin: 16px;
+  padding: 16px;
+  box-shadow: #0000001c 0px 0.3px 0.9px, rgba(0, 0, 0, 0.11) 0px 1.6px 3.6px;
+}
+
+.event-item div {
+  color: #78757f;
+  font-style: italic;
+}
+
+.event-item a {
+  position: absolute;
+  right: 16px;
+  bottom: 16px;
+  background-color: black;
+  color: white;
+  text-decoration: none;
+  padding: 4px 8px;
+  border-radius: 8px;
+}
+
+.event-item a:hover {
+  opacity: 0.8;
+}
+
+#upcoming-events-section .event-item a {
+  background-color: #2a9d8f;
+}
+
+#past-events-section .event-item a {
+  background-color: #8d99ae;
```

Comment:
I ran a color contrast check here too, and the white-on-gray has a color contrast ratio of **2.87:1**. This means that even if our text is larger, it would still not meet the required **3:1** color contrast ratio for large text. We should darken this gray somewhat, though how much darker it needs to be depends on whether you want to make the link text bigger.

Like with the upcoming event color contrast, I'm fine no matter how you want to handle this.